lib/skate/application.ex

```diff
@@ -29,11 +29,14 @@ defmodule Skate.Application do
           SkateWeb.Endpoint
         ]
 
-    link = Supervisor.start_link(children, strategy: :one_for_all, name: Skate.Supervisor)
+    case Supervisor.start_link(children, strategy: :one_for_all, name: Skate.Supervisor) do
+      {:ok, _pid} = link ->
+        Migrate.up()
+        link
```

Comment:
This is based on @firestack's observation that the migrator hides information in the case that `Supervisor.start_link` results in a failure. I didn't add Migrator as a child yet to limit the changes for now